ra2ce/analyses/indirect/origin_closest_destination.py

```diff
@@ -23,7 +23,7 @@
 # -*- coding: utf-8 -*-
 import copy
 import logging
-from typing import Optional, Union
+from typing import Optional, Union, Tuple, Any
```

Comment:
`Union` and `Tuple` are no longer needed as you can instead simple do:
- `a | b` instead of `Union[a, b]`
- `tuple[a, b]` instead of `Tuple[a,b]`